locales/en/kafka.json

```diff
@@ -122,6 +122,24 @@
   "partitions": "Partitions",
   "partitions_value": "up to 500",
   "select_filter_category": "Select filter category",
+  "settings": {
+    "connection_re_authentication_label": "Connection re-authentication.",
+    "Connection_re_authentication_states": {
+      "turning_off": "Turning off...",
+      "turning_on": "Turning on...",
+      "turnoff": "Off",
+      "turnon": "On"
+    },
+    "connection_re_authentication_switch": "Connection re-authentication.",
+    "connection_re_authentication_title": "Connection re-authentication:",
+    "error_alert_title": "Something went wrong",
+    "error_alert_title_description": "We're unable to update connection re-authentication at this time, Try again later.",
+    "kafka_instance_settings_label": "Configure global settings for this Kafka instance.",
+    "success_alert": "Connection re-authentication turned {{status}}",
```

Comment:
When this message displays, "Off" was displayed as the status. The `{{status}}` string that displays here should be lowercase instead. 

Also, I did not see a toast alert display when turning this on, but the setting also didn't update to show the on state. I'm not sure if that's an issue or just how the storybook is behaving.